.github/actions/setup-windows/action.yml

```diff
@@ -7,6 +7,8 @@ runs:
       run: |
         choco install mysql -y --no-progress --params="/port:3306"
         mysql.exe --port=3306 --user=root --password="" -e "ALTER USER 'root'@'localhost' IDENTIFIED BY 'Password12!'; FLUSH PRIVILEGES;"
+        mysql.exe --port=3306 --user=root --password="Password12!" -e "DROP DATABASE IF EXISTS test;"
```

Comment:
Then this will cost nothing. 😄 

I Get it, this script and the approach to maintaining them; are that the environment it happens to work in today, is great and will always be there and supported.

I Do not believe this to be the case.

Can you find me a case where this one-liner command will cause problems?

I Can tell you where problems will happen for anyone not using GitHub actions if they attempt to run a script more than once without a complete clean-room environment, which PHP core does not provide.